src/v/raft/state_machine.cc

```diff
@@ -144,7 +144,7 @@ ss::future<> state_machine::write_last_applied(model::offset o) {
     return _raft->write_last_applied(o);
 }
 
-ss::future<result<model::offset>> state_machine::instert_linerizable_barrier(
+ss::future<result<model::offset>> state_machine::insert_linerizable_barrier(
```

Comment:
```suggestion
ss::future<result<model::offset>> state_machine::insert_linerizable_barrier(
```

Reply:
nice catch. wanna send in another fix for this? @andrwng